playbooks/robusta_playbooks/events_enrichments.py

```diff
@@ -0,0 +1,56 @@
+from robusta.api import *
+
+
+class EventErrorReportParams(FindingKeyParams, RateLimitParams):
+    pass
+
+
+@action
```

Comment:
There are some weird couplings between this action and `event_resource_events`

1. The behaviour with `stop_playbook` is weird. I think we just want a rate_limit feature on the trigger-level (or playbook-level). This doesn't really belong in the `event_reporting`.

2. The behaviour with `FindingKeyParams` is non intuitive and creates a strange coupling between the two actions. Can we find a better way to do this? Couldn't `event_reporter` just set the default finding and then automatically all enrichments added later will be on that finding?

Reply:
1.
I don't think we want to rate limit on the trigger, but on the action.
A trigger can trigger many different actions, some may be rate limited, while others shouldn't be

2. I added the FindingKeyParams, to allow using this action in other playbooks as well.
Without it, the usage is very limiting, because you only allowed to add to the "default" finding.
This way, this is the default behavior, but you can override in case you need to

Reply:
fixed